### ISSUE TITLE
Traceback 

### ISSUE THREAD

**Issue by austinsonger** [TheresAFewConors/Sooty#58]
**Describe the bug**
Traceback

I've installed this tool many times and have only been able to get it working once and the last 4 times always fails. I work in a SOC and would love to be able to use this for my workflow.

**Reproduction Steps**

```
Traceback (most recent call last):
  File "Sooty.py", line 17, in <module>
    import strictyaml
ModuleNotFoundError: No module named 'strictyaml'
```

**Desktop Operating System:**
 - OS: Ubuntu 18.04, Ubuntu 20.04,  Windows Subsystem for Linux

**Comment by austinsonger** [comment]
I've had to installed the modules separately sometimes and it still doesn't work.

**Comment by TheresAFewConors** [comment]
This is related to using pip instead of pip3. Was addressed under #53
